### PR TITLE
Update 7.9.0 release notes

### PR DESCRIPTION
**Pull request by dliappis**
https://elasticsearch_61286.docs-preview.app.elstc.co/guide/en/elasticsearch/reference/7.9/release-notes-7.9.0.html

### CODE REVIEW

File: docs/reference/release-notes/7.9.asciidoc [hunk 1]
```diff
@@ -317,6 +317,10 @@ Search::
 * Don't omit empty arrays when filtering _source {es-pull}56527[#56527] (issues: {es-issue}20736[#20736], {es-issue}22593[#22593], {es-issue}23796[#23796])
 * Fix casting of scaled_float in sorts {es-pull}57207[#57207]
 
+Security::
+* Fix Elasticsearch field disclosure flaw (ESA-2020-12). (CVE-2020-7019). See
+https://discuss.elastic.co/t/elastic-stack-7-9-0-and-6-8-12-security-update/245456
```
Comment (dliappis) [inline docs/reference/release-notes/7.9.asciidoc:322]: @elastic/es-docs should this be something like this: https://discuss.elastic.co/t/elastic-stack-7-9-0-and-6-8-12-security-update/245456[annoucement in the Elastic forum] instead?

Comment (debadair) [reply docs/reference/release-notes/7.9.asciidoc:322]: In the past, we've included a link to the PR, and just linked the CVE number to the relevant info. See https://www.elastic.co/guide/en/elasticsearch/reference/7.9/release-notes-7.6.2.html for an example. (Not that it links to elastic.co/community/security, but it doesn't deep link to the specific CVE--linking to the forum post makes more sense to me.)

**Comment by lcawl** [comment]
I think this PR can be closed as a duplicate of https://github.com/elastic/elasticsearch/pull/61288

**Comment by dliappis** [comment]
Closing as discussed.

**Pull request closed without merging**